src/eventHandlers/ResourceBookingEventHandler.js

```diff
@@ -90,24 +92,24 @@ async function processUpdate (payload) {
     return
   }
   if (payload.value.status !== 'assigned') {
-    logger.info({
+    logger.debug({
       component: 'ResourceBookingEventHandler',
       context: 'processUpdate',
       message: `not interested resource booking - status: ${payload.value.status}`
     })
     return
   }
   const resourceBooking = await models.ResourceBooking.findById(payload.value.id)
-  const jobs = await models.Job.findAll({
-    where: {
-      projectId: resourceBooking.projectId,
-      deletedAt: null
-    }
-  })
-  for (const job of jobs) {
-    await selectJobCandidate(job.id, resourceBooking.userId)
-    await assignJob(job)
+  if (!resourceBooking.jobId) {
```

Comment:
Here checks whether `jobid` is missing in the `ResourceBooking` record. Simple exit the handler If there is not `jobId`.

Reply:
Also notice that logging was slightly improved in handlers so that we could focus on key information for the event handing.